ddtrace/tracer/tracer_test.go

```diff
@@ -301,6 +303,23 @@ func TestSamplingDecision(t *testing.T) {
 		child.Finish()
 		span.Finish()
 		assert.Equal(t, float64(ext.PriorityAutoReject), span.Metrics[keySamplingPriority])
+		assert.Equal(t, decisionKeep, span.context.trace.samplingDecision)
+	})
+
+	t.Run("dropped_stats", func(t *testing.T) {
+		tracer, _, _, stop := startTestTracer(t)
+		defer stop()
+		tracer.config.featureFlags = make(map[string]struct{})
```

Comment:
Will `featureFlags` be uninitialized by default for end-users? If so they'll see the same panic that showed up in the tests when `c.HasFeature` is called.

Reply:
`c.HasFeature` should still work since map *lookups* are OK on `nil` maps. You just can't assign to them like I did in the test.

Reply:
Got it, thanks!